app/assets/stylesheets/desktop/topic-post.scss

```
@@ -493,6 +493,20 @@ blockquote {
   padding: 12px;
 }
 
+aside.quote:has(.title) .title {
```

`:has` isn't necessary here

```suggestion
aside.quote .title {
```

---

doh